client/lib/posts/utils.js

```diff
@@ -151,10 +151,11 @@ var utils = {
 	},
 
 	normalizeAsync: function( post, callback ) {
+
 		postNormalizer(
 			post,
 			[
-				postNormalizer.waitForImagesToLoad,
+				//postNormalizer.waitForImagesToLoad,
```

Comment:
really we can kill the "async" rules all together and move keepValid over to the sync ruleset.